src/main/java/Duke.java

```diff
@@ -14,30 +14,52 @@ public static void main(String[] args) {
         while (!line.equals("bye")){
             if(line.equals("list")){
                 System.out.println(HORIZONTAL_LINE);
+                System.out.println("  Here are the tasks in your list:");
                 for (int i = 0; i < tasks.size(); i++) {
-
-                    System.out.println((i+1)+":"+"["+tasks.get(i).getStatusIcon()+"] "+tasks.get(i).description);
+                    System.out.println("  " + (i+1) +":" + tasks.get(i));
                 }
                 System.out.println(HORIZONTAL_LINE);
             }else if(line.startsWith("done")){
                 System.out.println(HORIZONTAL_LINE);
                 int taskNumberCompleted = Integer.parseInt(line.replaceAll("\\D+",""))-1;
                 tasks.set(taskNumberCompleted,tasks.get(taskNumberCompleted).completeTask());
-                System.out.println(" Nice! I've marked this task as done:");
-                System.out.println(" " + "["+tasks.get(taskNumberCompleted).getStatusIcon()+"] "+tasks.get(taskNumberCompleted).description);
+                System.out.println("  Nice! I've marked this task as done:");
+                System.out.println("   "+":"+tasks.get(taskNumberCompleted));
                 System.out.println(HORIZONTAL_LINE);
                 System.out.println();
-            } else{
+            } else if(line.startsWith("todo")){
+                System.out.println(HORIZONTAL_LINE);
+                System.out.println("  Got it. I've added this task:");
+                tasks.add(new ToDo(line));
+                System.out.println("  " + new ToDo(line));
+                System.out.println("  Now you have " + tasks.size() + " tasks in the list.");
+                System.out.println(HORIZONTAL_LINE);
+            } else if(line.startsWith("deadline")){
+                System.out.println(HORIZONTAL_LINE);
+                System.out.println("  Got it. I've added this task:");
+                tasks.add(new Deadline(line));
+                System.out.println("  " + new Deadline(line));
+                System.out.println("  Now you have " + tasks.size() + " tasks in the list.");
+                System.out.println(HORIZONTAL_LINE);
+            }
+            else if(line.startsWith("event")){
+                System.out.println(HORIZONTAL_LINE);
+                System.out.println("  Got it. I've added this task:");
+                tasks.add(new Event(line));
+                System.out.println("  " + new Event(line));
+                System.out.println("  Now you have " + tasks.size() + " tasks in the list.");
+                System.out.println(HORIZONTAL_LINE);
+            }
+            else {
                 System.out.println(HORIZONTAL_LINE);
-                System.out.println("  added :" + line);
-                tasks.add(new Task(line));
+                System.out.println(" Invalid input." + System.lineSeparator() + " Type bye to exit.");
```

Comment:
instead of putting so many print lines under one condition, extract them and name them according to their commands